Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -766,11 +860,14 @@ CGMutablePathRef CGPathCreateMutableCopyByTransformingPath(CGPathRef path, const
 
     if (transform && !CGAffineTransformEqualToTransform(*transform, CGAffineTransformIdentity)) {
         CGMutablePathRef transformedPath = CGPathCreateMutable();
-        path->ClosePath();
+        FAIL_FAST_IF_FAILED(path->ClosePath());
+
+        transformedPath->SetStartingPoint(CGPointApplyAffineTransform(path->GetStartingPoint(), *transform));
 
+        transformedPath->BeginFigure();
```

Comment:
>BeginFigure [](start = 25, length = 11)

I wonder what happens if you take a mutable copy of a path with an open figure.. hm